test/unit/helpers/xhr/MFA_ENROLL_allFactors_OnPrem.js

```diff
@@ -249,9 +268,6 @@ define({
                 ]
              }
           }
-       },
-       "profile":{
```

Comment:
Do we not need profile for claims_provider?

Reply:
Yeah, we don't have a strong case for the profile. So the server side is not setting it for claims_provider.